meta-lmp-support/recipes-bsp/u-boot/u-boot-fio_%.bbappend

```diff
@@ -1,3 +1,11 @@
 FILESEXTRAPATHS_prepend_imx8mmevk := "${THISDIR}/${PN}:"
 
 SRC_URI_append_imx8mmevk += "file://0001-Disable-1V8-on-usdhc2-for-SDMUX.patch "
+
+do_configure_append_imx8mmevk() {
+    # boot from SD
```

Comment:
Is it possible to put this behind a configuration flag instead of target? I'm asking as we support SD card by default also for Hummingboard targets (see https://github.com/PelionIoT/meta-pelion-edge/pull/300).

Reply:
This could be generalized e.g. in `local.conf` once it has been tested to work on some other MACHINEs in addition to imx8mmevk?

Reply:
Yep. Leave it is now and let's see when we continue work on #300